Libs/src/main/java/com/nearsoft/products/libs/model/Product.java

```diff
@@ -0,0 +1,38 @@
+package com.nearsoft.products.libs.model;
+
+import lombok.*;
```

Comment:
Avoid using wildcards import :) if this is added by IDE there is a config section to change this